### PR TITLE
My submission for the weekly challenge

### PR DESCRIPTION
**Pull request by hdck007**
I have created two scripts that change your wallpaper on ubuntu and termux(for android).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,18 @@ cd week-3
...
+**6.** Make a new directory with your Full_Name.
+```terminal
+mkdir FirstName_LastName
+```
+**6.1** Create a file wallpaperChanger.py
+```terminal
+touch wallpaperChanger.py
+```
 
+<<<<<<< HEAD
```
Comment (meta-boy) [inline README.md]: Remove merge conflicts

**Review by meta-boy — CHANGES_REQUESTED** [review]
Remove merge conflicts

**Comment by hdck007** [comment]
I have added new commits to solve the merge conflicts.

**Review by meta-boy — APPROVED** [review]
LGTM

**Pull request merged**